code/kg2/filter_kg_and_remap_predicates.py

```diff
@@ -151,4 +152,11 @@ def make_arg_parser():
             print('relation curie is in the config file but was not detected in the graph: ' + relation_curie, file=sys.stderr)
     for relation_curie in relation_curies_not_in_nodes:
         print('could not get IRI for relation curie: ' + relation_curie)
+    build_info = {
+        "name" : "KG2:Build",
+        "date" : datetime.now().strftime("%m-%d-%Y"),
+        "nodes" : len(graph["nodes"]),
+        "edges" : len(graph["edges"]),
+    }
+    graph["build"] = build_info
```

Comment:
added a whole top level slot for build info instead of including it as a node or an edge: hopefully that doesn't mess with anything too far down the line. 
`build-kg2.sh test` completes without error on kg2dev, but that is the extent of my testing :)

Reply:
Thanks. I think having the build info as a top-level slot is a good idea.

Howeve, the issue here is that if the build info isn't included as a node, it won't get into Neo4j. Can you please also include it as a node?  

Also, can you please update `kg2_slim.py` to preserve the `build` slot?  As it is written, `kg2_slim.py` will not preserve it (I think). Thank you!

Reply:
@saramsey For the build node, what should "category label" be set to?

Reply:
Hi @kvarforl for the biolink `category label` for the build node, how about `information content entity`?

https://biolink.github.io/biolink-model/docs/InformationContentEntity.html